lib/Utils/AffineMapUtils.h

```diff
@@ -0,0 +1,31 @@
+#ifndef LIB_UTILS_AFFINEMAPUTILS_H_
+#define LIB_UTILS_AFFINEMAPUTILS_H_
+
+#include "mlir/include/mlir/IR/AffineMap.h"  // from @llvm-project
+#include "mlir/include/mlir/Support/LLVM.h"  // from @llvm-project
+
+namespace mlir {
+namespace heir {
+
+// Construct an identity permutation of size n
+inline SmallVector<int64_t> identity(int64_t n) {
+  SmallVector<int64_t> permutation;
+  for (int64_t i = 0; i < n; i++) {
+    permutation.push_back(i);
+  }
```

Comment:
i learned about std::iota recently which you could use here but i don't mind because clear code is clear.

Reply:
TIL!